UnityProject/Assets/Scripts/Items/Cards/Emag.cs

```diff
@@ -89,6 +90,11 @@ public bool EmagHasCharges()
 		/// </summary>
 		public bool UseCharge(HandApply interaction)
 		{
+			AdminLogsManager.AddNewLog(
+				interaction.Performer,
+				$"{interaction.PerformerPlayerScript.visibleName} has emmaged {interaction.TargetObject.name}.",
+				LogCategory.Interaction,
+				Severity.ANNOYING);
```

Comment:
I don't think it's that time of severity since sometime security/cargo grabs it and use it a bunch for free stuff

Reply:
nu uh